test/backend/datastores.livy.spec.js

```diff
@@ -76,7 +76,8 @@ df.registerTempTable("ALCOHOL_CONSUMPTION_BY_COUNTRY_2010")
             'ALCOHOL_CONSUMPTION_BY_COUNTRY_2010';
 
         return query(`SELECT * FROM ${tableName}\nLIMIT 5`, connection).then(results => {
-            assert.deepEqual(results.rows, [
+            const roundedToSinglePrecision = results.rows.map(r => [r[0], Number(r[1].toPrecision(6))]);
```

Comment:
It looks like when the table was created in Spark, the source CSV file was imported as single-precision numbers. But when we query Spark/Livy, the results are reported as double-precision numbers.

@chriddyp Have you come across this issue with any of the other connectors?

Reply:
Hm, I haven't come across this although I haven't really investigated too deeply either

Reply:
@tarzzz @chriddyp Here's an untested explanation: I've just noticed that [the script to create the sample Hive table declares the column as `FLOAT`](https://github.com/plotly/plotly-database-connector/blob/2eae53de7347051328ea3cd6bd428609d765a90d/test/docker/hadoop-spark/create_hive_tables.py#L14).